devtools-ci-index.yaml

```diff
@@ -4470,6 +4470,12 @@
             saas_git: saas-openshiftio
             timeout: '10m'
             extra_target: rhel
+        - '{ci_project}-{git_repo}-nightly':
+            git_organization: eclipse
+            git_repo: che-plugin-registry
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_nightly.sh'
+            timeout: '10m'
```

Comment:
Any specific reason why decreasing timeout from 30 to 10 minutes? I can imagine, that docker build&push could in some circumstances take more than that.

Reply:
What annoyed me was the inconsistency with the CI build above https://github.com/openshiftio/openshiftio-cico-jobs/pull/1043/files#diff-a0fd432ddaac45438209cf6e0a7298c0R4471.

Reply:
But in the meantime we have introduced the patched images build in the nightly script so I guess I understand now why @ibuziuk set 30 min here. I revert this change and will update the `build-master` job as well (one day we may decide to run the patched images build on the `build-master`)